conans/test/command/search_test.py

```diff
@@ -193,6 +195,8 @@ def package_search_properties_filter_test(self):
             arch: x86
             compiler: gcc
             compiler.version: 4.3
+        [recipe hash]
```

Comment:
I thought that it could be useful to compare between generated packages. But it's not very important.